### DISCUSSION TITLE
Ascii art for "Directus" or the rabbit logo, or both!

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13345] [Feature Requests]
Wouldn't it be cool to have some ascii art of either 'Directus' or the rabbit logo in the console when directus starts and as headers in some text files.

## figlet converts text to ascii art
- `sudo apt install figlet; showfigfonts; figlet Directus; figlet -f _any of the font names from showfigfonts_ Directus` 
    - `showfigfonts` to see a list of existing fonts, loads more fonts can be installed from http://www.figlet.org/fontdb.cgi 
    - https://manpages.ubuntu.com/manpages/bionic/man6/figlet-figlet.6.html
    - https://devhints.io/figlet - cheatsheet

`figlet Directus`
```
 ____  _               _             
|  _ \(_)_ __ ___  ___| |_ _   _ ___ 
| | | | | '__/ _ \/ __| __| | | / __|
| |_| | | | |  __/ (__| |_| |_| \__ \
|____/|_|_|  \___|\___|\__|\__,_|___/

```
`figlet -f slant Directus`
```
    ____  _                __            
   / __ \(_)_______  _____/ /___  _______
  / / / / / ___/ _ \/ ___/ __/ / / / ___/
 / /_/ / / /  /  __/ /__/ /_/ /_/ (__  ) 
/_____/_/_/   \___/\___/\__/\__,_/____/  
```

## converting images to ascii art
- `sudo apt install jp2a; jp2a someimage.jpg` 
- https://www.linuxfordevices.com/tutorials/linux/images-to-ascii-art

**Comment by benhaynes** [comment]
But... we have one in the App's dev tools. I defer to @rijkvanzanten on if we want to include this in the console.

<img width="601" alt="Screen Shot 2022-05-17 at 8 37 24 AM" src="https://user-images.githubusercontent.com/522079/168824115-a2717de8-bee8-4fcf-bcc3-f9cebcc2d32b.png">

**Reply by sxsws** [reply]
that's great!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
